src/modules/taskFunctions.js

```diff
@@ -2,9 +2,10 @@
 export function saveTasksToLocalStorage(taskList) {
   const tasks = [];
   const taskItems = taskList.querySelectorAll('li');
-  taskItems.forEach((taskItem, index) => {
-    const text = taskItem.querySelector('span').textContent;
-    tasks.push({ index: index + 1, text });
+  taskItems.forEach((taskItem) => {
+    const text = taskItem.querySelector('.task-item span').textContent;
+    const completed = taskItem.querySelector('.task-checkbox').checked;
+    tasks.push({ text, completed });
```

Comment:
- [ ] Good job implementing requested changes but now tasks don't have indexes, each task should have an index and this is part of the project but also make sure that after deleting tasks the index should be updated also for the remaining tasks, Please fix this 🙏🏾 

<img width="1410" alt="Screenshot 2023-08-26 at 17 19 27" src="https://github.com/DevKaranJ/ToDo-list/assets/74501523/d43a42a9-d3e1-4247-bf5e-95cb00ee8b8b">